client/state/themes/selectors.js

```diff
@@ -604,7 +604,7 @@ export function isThemePremium( state, themeId ) {
 export function isPremiumThemeAvailable( state, themeId, siteId ) {
 	return isThemePurchased( state, themeId, siteId ) || (
 			hasFeature( state, siteId, FEATURE_UNLIMITED_PREMIUM_THEMES ) &&
-			( ! isJetpackSite( state, siteId ) || config.isEnabled( 'jetpack/pijp' ) )
+			! isJetpackSite( state, siteId )
```

Comment:
Since this was previously an 'or' (`||`) check, I think we need to remove the entire line. Rationale: We could say `config.isEnabled( 'jetpack/pijp' )` is now always `true`. Then so is `anything || config.isEnabled( 'jetpack/pijp' )` (which amounts to `anything || true`).